src/components/Table.jsx

```diff
@@ -27,6 +27,9 @@ import useSessionStorage from "../hooks/useSessionStorage";
 import { status } from "../constants/index";
 import AssignToEditor from "../logos/AssignToEditor.svg";
 import AssignToWriter from "../logos/AssignToWriter.svg";
+import ArrowSortDownLine from "../logos/ArrowSortDownLine.svg";
+import ArrowSortUpLine from "../logos/ArrowSortUpLine.svg";
+import { Tooltip } from 'primereact/tooltip';
```

Comment:
Import from libraries should be at top ahead of file imports.

Follow same standards everywhere.

Reply:
addressed